### PR TITLE
Adding mofo-style and fixing bustifications

### PR DESCRIPTION
**Pull request by simonwex**
# In progress

_Outstanding_
* [x] Fix style issues
* [x] fix outstanding bugs

### CODE REVIEW

**Comment by simonwex** [comment]
I think this last one takes care of tests. 

`/me waits for Travis`

File: lib/account.js.b [hunk 1]
```diff
@@ -0,0 +1,59 @@
+var hyperquest = require('hyperquest');
```
Comment (alicoding) [inline lib/account.js.b]: hmm what is this `.js.b` ?

Comment (simonwex) [inline lib/account.js.b]: Ah, an artifact that shouldn't be there. Good catch.

**Comment by cadecairos** [comment]
R+

**Pull request merged**